js/ui/filter_builder/filter_builder.js

```diff
@@ -587,7 +587,7 @@ var FilterBuilder = Widget.inherit({
     },
 
     _initCustomOperations: function() {
-        this._customOperations = utils.getMergedOperations(this.option("customOperations"), this.option("filterOperationDescriptions.between"));
+        this._customOperations = utils.getMergedOperations.bind(this)(this.option("customOperations"), this.option("filterOperationDescriptions.between"));
```

Comment:
Why? The getMergedOperations method is static and it does not use this keyword. So it is very strange to bind it to this.

Reply:
Looks like hack to bind editorTemplate to this. It will be batter pass this as parameter to getMergedOperations method.